record_screenshots.sh

```diff
@@ -0,0 +1,8 @@
+set -e
+
```

Comment:
You can add this scenario to makefile. It's a common place.

Reply:
removed